src/Components/Button/ButtonLikeDown.tsx

```diff
@@ -0,0 +1,6 @@
+import React from 'react'
+
+
+export const ButtonLikeDown = ({img}: any) => {
+    return <button className='ButtonlikeDown'><img className='ImglikeDown' src={img}/></button>
+}
```

Comment:
Если у тебя частный компонент как этот. То img у тебя всегда один и тот же, поэтому нужно его не пробрасывать через пропсы, а назначать сразу же в src. 
То же самое относиться и к компонентам Dots и другим кнопкам с иконками
```suggestion
import likeDownIcon from './somePathToIcon'
export const ButtonLikeDown = ({img}: any) => {
    return <button className='ButtonlikeDown'><img className='ImglikeDown' src={likeDownIcon}/></button>
}
```